scrunch/datasets.py

```diff
@@ -2371,15 +2371,23 @@ def fork(self, description=None, name=None, is_published=False,
             dataset otherwise the owner will be the current user in the
             session and the Dataset will be set under `Personal Project`.
         :param project: str, default=None
-           The project ID or URL for the project in which the fork 
-           dataset should be created.
+            The project ID or URL for the project in which the fork
```

Comment:
To make the behavior of project clearer, we should add to the docstring an explanation of what happens when project is None.

Reply:
I've updated the docstring for the `project` & `preserve_owner` parameters.

Reply:
Thanks @shaikh-ma 🙇